docs/mechanical/fabrication_Engineer.md

```diff
@@ -1,3 +1,15 @@
-# Fabrication Engineer
+---
+title: Role of Fabrication Engineer
+author: Shubham Singh
+email: shubhamsingh2454@gmail.com
+---
 
-1. Location of acquisition of materials - Each material, exact location, cost with date.
+- Should be able to study plans for upcoming design plans, and develop strategies for improving upon current models.
+- Should know AUTOCAD & SolidWorks and should be able to develop plans from the 3D design.
+- Develop and build prototypes and run tests to measure their level of function.
+- Record and evaluate testing data, altering designs as necessary to bring them to safety, performance and efficiency standards.
+- The fabrication engineer is in charge of testing raw materials. He should follow a certain set of tests to identify the strength and durability of the materials to be used in the production. He may also devise and develop quality standards for future projects and reference of other staff in the organization.
+- Consult with other teams during product manufacturing, advising them on design specifications and providing physical assistance when required.
+- Calculate cost estimates for final product designs, and release reports to supervisors, incorporating costs of labour, material, delivery and overhead.
+- Attend weekly team meetings to provide progress reports, obtain updates on colleagues’ progress, strategize on steps for the following week and acquire information about projected emerging product lines.
```

Comment:
Please look at this typo!

Reply:
It is not a typo. [https://www.merriam-webster.com/dictionary/strategize](https://www.merriam-webster.com/dictionary/strategize).
Don't know how github analyzes typos.